CoreQualityControlView/src/au/gov/asd/tac/constellation/views/qualitycontrol/daemon/QualityControlAutoVetter.java

```diff
@@ -152,10 +156,25 @@ public void graphChanged(final GraphChangeEvent event) {
         if (graph != null) {
             final ReadableGraph readableGraph = graph.getReadableGraph();
             try {
+                final int cameraAttribute = VisualConcept.GraphAttribute.CAMERA.get(readableGraph);
+                final int vxselectedAttribute = VisualConcept.VertexAttribute.SELECTED.get(readableGraph);
+                final int txselectedAttribute = VisualConcept.VertexAttribute.SELECTED.get(readableGraph);
+
                 final long thisGlobalModificationCounter = readableGraph.getGlobalModificationCounter();
+                final long thisCameraModificationCounter = readableGraph.getValueModificationCounter(cameraAttribute);
+                final long thisvxselectedAttributeModificationCounter = readableGraph.getValueModificationCounter(vxselectedAttribute);
+                final long thistxselectedAttributeModificationCounter = readableGraph.getValueModificationCounter(txselectedAttribute);
+
                 if (thisGlobalModificationCounter != lastGlobalModificationCounter) {
-                    updateQualityControlState(graph);
+                    if (lastCameraModificationCounter == thisCameraModificationCounter
+                            && lastvxselectedAttributeModificationCounter == thisvxselectedAttributeModificationCounter
+                            && lasttxselectedAttributeModificationCounter == thistxselectedAttributeModificationCounter) {
```

Comment:
We will want the quality control state to update with selection changes so it would be good to remove these 2 conditions (and consequently can probably remove `thisvxselectedAttributeModificationCounter` and `thistxselectedAttributeModificationCounter` altogether)